digits/utils/image.py

```diff
@@ -11,6 +11,20 @@
 except ImportError:
     from StringIO import StringIO
 
+try:
```

Comment:
so you can add `pydicom` to `requirements.txt` and `import dicom` without a `try/except` then?